dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1025,3 +1025,19 @@ def test_writing_parquet_with_unknown_kwargs(tmpdir, engine):
 
     with pytest.raises(TypeError):
         ddf.to_parquet(fn,  engine=engine, unknown_key='unknown_value')
+
+
+def test_setect_partitioned_column(tmpdir, engine):
+    if engine == 'pyarrow':
+        pytest.xfail()
```

Comment:
@jcrist , I don't know if you know anything about partition reading in pyarrow parquet. Perhaps can ping someone else - this test fails for engine==pyarrow.

Reply:
This actually a bug inside `pyarrow`. I'll take care of this, we should be able to activate this test then with `pyarrow==0.9.0`.

Reply:
Yep, would be happy to see this replaced by an xfail depending on pyarrow's version.

Reply:
Here it is: https://github.com/dask/dask/pull/3207